staging/src/k8s.io/legacy-cloud-providers/aws/aws_fakes.go

```diff
@@ -488,12 +488,6 @@ func (elb *FakeELB) ModifyLoadBalancerAttributes(*elb.ModifyLoadBalancerAttribut
 	panic("Not implemented")
 }
 
-// expectDescribeLoadBalancers is not implemented but is required for interface
-// conformance
```

Comment:
Is this comment accurate? Is there an interface that requires this?

Reply:
> Is this comment accurate? Is there an interface that requires this?

There is currently no interface to use this function, only in aws_test.go this function is used for testing.Do I need to delete the corresponding function in the test file.